packages/semantic-ui/src/SelectWidget/SelectWidget.js

```diff
@@ -78,7 +79,18 @@ function SelectWidget(props) {
     onBlur,
     onFocus,
   } = props;
-  const semanticProps = getSemanticProps({ options });
+  const semanticProps = getSemanticProps({
+    schema,
+    uiSchema,
+    formContext,
+    options,
+    defaultSchemaProps:{
+      inverted: "false",
```

Comment:
Is making the `inverted` value a string (vs it being a boolean everywhere else) necessary? Consider switching it and seeing if things still work

Reply:
the react props warn for the select widget it not being a string. strange I know